changelog/unreleased/pull-4692

```diff
@@ -0,0 +1 @@
+feat: dump flag --target should be allowed to write existing file
```

Comment:
Please update the changelog added by https://github.com/restic/restic/pull/4682/files .

Reply:
@MichaelEischer  you means `changelog/unreleased/issue-4678`?  Can you tell me what should I do, this is my first pr

Reply:
I've combined both changelog entries. The `--target` option is not yet part of a released version. Thus, there should only be a single changelog entry that describes that option. Otherwise, we'd end up with partially contradictory changelog entries.